### DISCUSSION TITLE
自修了一些源

### DISCUSSION THREAD

**Discussion by falling7down** [xiaohucode/yidaRule#211] [Show and tell]
[下载后本地导入](https://share.lanol.cn/#/?code=19763)

**Comment by Luke2084** [comment]
> [限制下载10次](https://share.lanol.cn/#/?code=20787)
> 
> 
> 
> 

感谢🙏您，又可以用益达看动漫了🥹

**Reply by zy272425559** [reply]
大佬，下载不了。能重新分项下吗？

**Comment by ApolloRioo** [comment]
https://www4.bestjavporn.com/zh/

大佬，能写一下这个网站吗？感谢有你。

**Reply by falling7down** [reply]
> https://www4.bestjavporn.com/zh/
> 大佬，能写一下这个网站吗？感谢有你。

不会解密视频地址，等开发者写吧

**Reply by ApolloRioo** [reply]
> > https://www4.bestjavporn.com/zh/
> > 大佬，能写一下这个网站吗？感谢有你。
> 
> 不会解密视频地址，等开发者写吧
> 

开发者消失多年了😣

**Reply by 8721653** [reply]
大佬可以重新发一下嘛

**Comment by Oldsport-996** [comment]
楼主，您知道本地怎么导入吗，我开了加速器也没法导入规则，也没找到本地导入的入口

**Reply by falling7down** [reply]
底栏“我的”>管理站点>更多>本地导入

**Comment by ywjang5** [comment]
https://m.or77.net/
大佬，能写一下这个网站吗？麻烦大大了

**Comment by wuhao1115** [comment]
大佬文件过期了，下载不了，能重新发一下吗

**Comment by laoling123** [comment]
大佬，显示文件不存在_(:з」∠)_

**Comment by suite2018** [comment]
大佬，无法下载呢，能不能帮忙把爱听书的源修改一下，无法显示第一页的后面内容

**Comment by 8721653** [comment]
> [下载后本地导入](https://share.lanol.cn/#/?code=19763)请问取件码是什么谢谢

**Comment by w151729** [comment]
大佬可以再分享一次吗
